test/e2e/scheduling/preemption.go

```diff
@@ -75,6 +76,10 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 		for _, pair := range priorityPairs {
 			cs.SchedulingV1().PriorityClasses().Delete(context.TODO(), pair.name, *metav1.NewDeleteOptions(0))
 		}
+		for _, node := range nodeList.Items{
```

Comment:
```suggestion
		for _, node := range nodeList.Items {
```

Reply:
Ah yeah, this would fail gofmt check anyway I think